docker-compose.yml

```diff
@@ -58,7 +58,7 @@ services:
       - RELEASE_BUMPER=foobar
 
   diagnostics:
-    image: nebraltd/hm-diag:b1210ec
+    image: nebraltd/hm-diag:7e26f35
     environment:
       - FIRMWARE_VERSION=2021.11.17.2-1
```

Comment:
Also need to bump FIRMWARE_VERSION for hm-config and hm-diag.

Reply:
Should I update to FIRMWARE_VERSION=2021.11.17.2-2? or FIRMWARE_VERSION=2021.11.17.3-1?

Reply:
2021.11.19.1? Sorry didn't realize it was the date at first 😂

Reply:
`2021.11.17.2-2`. The `2021.11.17.2` part is determined by the [most recent GA on hm-miner](https://github.com/NebraLtd/hm-miner/commit/b34655ff2c59a0ad00546d3339321cab336bc030). `-3` at the end is incremented each time we make a new release with the same GA.

Reply:
> 2021.11.19.1? Sorry didn't realize it was the date at first 

No worries. This is not well documented and we've been inconsistent about it in the past. That date is the date the last GA was released, which will not always correspond to additional releases we make on top of the GA.

Reply:
Ohhh ok that is more clear now, thank you! I bumped it up, should be good now